crates/collateral/src/lib.rs

```diff
@@ -165,6 +165,29 @@ impl<T: Config> Module<T> {
         Ok(())
     }
 
+    /// Transfer DOT from a sender a receiver. Can only fail if
```

Comment:
I don't see how this is different from transfer? Is the difference that we transfer from locked to locked? Please add as comment here

Reply:
Yes, that's the difference